wdl-lint/src/rules/malformed_lint_directive.rs

```diff
@@ -0,0 +1,137 @@
+//! A lint rule for flagging malformed lint directives.
+use wdl_ast::AstToken;
+use wdl_ast::Comment;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Document;
+use wdl_ast::Span;
+use wdl_ast::SupportedVersion;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+use crate::util::is_inline_comment;
+
+/// The identifier for the Malformed Lint Directive rule.
+const ID: &str = "MalformedLintDirective";
+/// The accepted lint directives.
+const ACCEPTED_LINT_DIRECTIVES: [&str; 1] = ["except"];
+
+/// Creates an "Excessive Whitespace" diagnostic.
+fn excessive_whitespace(span: Span) -> Diagnostic {
+    Diagnostic::warning("expected exactly one space before lint directive")
+        .with_rule(ID)
+        .with_label("excessive whitespace", span)
+}
+
+/// Creates an "Inline Lint Directive" diagnostic.
+fn inline_lint_directive(span: Span) -> Diagnostic {
+    Diagnostic::warning("lint directive must be on its own line")
+        .with_rule(ID)
+        .with_label("inlined lint directive", span)
+}
+
+/// Creates an "Invalid Lint Directive" diagnostic.
+fn invalid_lint_directive(span: Span) -> Diagnostic {
+    let accepted_directives = ACCEPTED_LINT_DIRECTIVES.join(", ");
+    Diagnostic::warning("lint directive not recognized")
```

Comment:
Can we include the lint directive name in the diagnostic's message?

```suggestion
    Diagnostic::warning(format!("lint directive `{name}` is not recognized"))
```